### DISCUSSION TITLE
I wish to make a simple android app using paddleocr

### DISCUSSION THREAD

**Discussion by kishore-work-hard** [PaddlePaddle/PaddleOCR#13600] [Ideas & Features]
I want to run this paddle ocr model in android app. flutter or android studio or using python for android. how to achieve this? i dont want to do api calls. i just want to run paddle ocr model in androd phone locally via an app. is it possible? if possible please give me an document guide.

**Comment by SWHL** [comment]
https://github.com/PaddlePaddle/PaddleOCR/tree/main/deploy/android_demo

**Reply by kishore-work-hard** [reply]
> https://github.com/PaddlePaddle/PaddleOCR/tree/main/deploy/android_demo

is it possible to do the same using python? like make an android app using python?

**Reply by SWHL** [reply]
I'm not sure about that, you need to ask other friends online

**Comment by kishore-work-hard** [comment]
oh man. I need to learn android studio from scratch to do this. it is
android studio -java right?
